javascript/builtins/Error.json

```diff
@@ -228,6 +228,56 @@
             }
           }
         },
+        "captureStackTrace": {
+          "__compat": {
+            "tags": [
+              "web-features:javascript"
+            ],
+            "support": {
+              "chrome": {
+                "version_added": "17"
+              },
+              "chrome_android": "mirror",
+              "deno": {
+                "version_added": false
+              },
+              "edge": "mirror",
+              "firefox": {
+                "version_added": "136",
+                "flags": [
+                  {
+                    "type": "preference",
+                    "name": "javascript.options.experimental.error_capture_stack_trace",
+                    "value_to_set": "true"
+                  }
+                ],
+                "impl_url": "https://bugzil.la/1886820"
+              },
+              "firefox_android": "mirror",
+              "ie": {
+                "version_added": false
+              },
+              "nodejs": {
+                "version_added": "16.17.0"
```

Comment:
This shows up in node docs in https://nodejs.org/docs/latest-v16.x/api/errors.html#errorcapturestacktracetargetobject-constructoropt (16.20.2) but that isn't an option. It isn't in the 15.x release docs before that.